src/components/form-tags/form-tags.js

```diff
@@ -42,23 +42,18 @@ const cleanTags = tags => {
 // Processes an input/change event, normalizing string or event argument
 const processEventValue = evt => (isString(evt) ? evt : isEvent(evt) ? evt.target.value || '' : '')
 
-// Returns a ffresh empty tagsState object
-const cleanTagsState = () => {
-  return {
-    all: [],
-    valid: [],
-    invalid: [],
-    duplicate: []
-  }
-}
+// Returns a fresh empty `tagsState` object
+const cleanTagsState = () => ({
+  all: [],
+  valid: [],
+  invalid: [],
+  duplicate: []
+})
+
 // @vue/component
 export const BFormTags = /*#__PURE__*/ Vue.extend({
   name: NAME,
   mixins: [idMixin, normalizeSlotMixin],
-  model: {
```

Comment:
The model is needed so that the Docs will properly reflect which prop/event is used for the v-model.